### PR TITLE
Add a warning about the automated workflow to the end of the talk checklist issue

### PR DESCRIPTION
**Pull request by sgibson91**
### Summary

This just adds a warning that these issues can be closed by an automated workflow and that it's ok to reopen them if needed.

### List of changes proposed in this PR (pull-request)

What it looks like rendered

Screenshot 2022-08-24 at 18 46 19

### What should a reviewer concentrate their feedback on?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/give_a_talk.yml [hunk 1]
```diff
@@ -30,8 +30,16 @@ body:
...
+        - label: If you re-used slides from a specific talk, please acknowledge the original author of the slides
         - label: Double check acknowledgements slide for TTW team, license info, Scriberia link, any additional acknowledgements
         - label: Double check contact info for TTW links (book, Twitter, GitHub, Slack, newsletter), your own contact into
         - label: Share the Zenodo link and a recording (if available) in TTW Slack!
- 
+  - type: markdown
+    attributes:
+      value: |
+        ---
+        :warning: _Warning_ :warning:
```
Comment (aleesteele) [inline .github/ISSUE_TEMPLATE/give_a_talk.yml:41]: Good call! This is a great idea to add the warning, to catch no-body off guard.

File: .github/ISSUE_TEMPLATE/give_a_talk.yml [hunk 2]
```diff
@@ -30,8 +30,16 @@ body:
...
+
+        _This issue will be automatically closed by a GitHub Actions workflow when more_
```
Comment (aleesteele) [inline .github/ISSUE_TEMPLATE/give_a_talk.yml:43]: Maybe you can add a link here to a guide or resource about Github actions, or a short description? Many beginners might not understand what they are.

**Review by aleesteele — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/give_a_talk.yml (hunk 2 above)
Comment (sgibson91) [inline .github/ISSUE_TEMPLATE/give_a_talk.yml:43]:
```suggestion
        _This issue will be automatically closed by a [GitHub Actions](https://docs.github.com/en/actions) workflow when more_
```

Comment (sgibson91) [reply .github/ISSUE_TEMPLATE/give_a_talk.yml]: Done in [`63853f5` (#2603)](https://github.com/alan-turing-institute/the-turing-way/pull/2603/commits/63853f530a03234f65f8bc9e93d7a6c022035b8f)

**Review by aldenc — APPROVED** [review]
looks good!

**Pull request merged**